### DISCUSSION TITLE
please add Icon to MultiSelect component

### DISCUSSION THREAD

**Discussion by anna777x** [dotnetcore/BootstrapBlazor#3402] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

I'm trying to set icon in MultiSelect component

### Describe the solution you'd like

Please add Icon to MultiSelect component like you have in Select Component.

**Comment by bb-auto** [comment]
@anna777x Thank you for contacting us. We will give feedback later.

**Reply by ArgoZhang** [reply]
@anna777x what icon?

**Reply by anna777x** [reply]
Expand icon, like you have on any select components

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/61416594/3d904400-ce17-4a74-b633-284d8c1c902e)

**Comment by ArgoZhang** [comment]
There is no place for this icon

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/344afc91-bbfd-4c7c-86b2-8d5786c82106)

**Reply by anna777x** [reply]
just idea: if you add more items to selected list, scroll appears on the right side, so we have place. Maybe we just need to show icon OR scroll depends on control's state.
do you understand my suggestion?
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/61416594/7bb9bd66-3562-4f9b-a118-e5d2e977eee5)

**Reply by ArgoZhang** [reply]
thanks for your suggestion. yes, I completely understand your suggestion. After adding more options, regardless of whether there is a scrollbar or not, this icon looks very inappropriate and wastes some space for it.

any way good suggestion
